config/assets/default.js

```diff
@@ -14,160 +14,160 @@ module.exports = {
-        'public/lib/mailcheck/src/mailcheck.js',
-        'public/lib/angular-mailcheck/angular-mailcheck.js',
-        'public/lib/angular-locker/dist/angular-locker.js',
-        'public/lib/angular-confirm-modal/angular-confirm.js',
-        'public/lib/angulargrid/angulargrid.js',
-        'public/lib/firebase/firebase-app.js',
-        'public/lib/firebase/firebase-messaging.js'
+        'node_modules/jquery/dist/jquery.js',
+        'node_modules/angular/angular.js',
+        'node_modules/angular-aria/angular-aria.js',
+        'node_modules/angular-resource/angular-resource.js',
+        'node_modules/angular-animate/angular-animate.js',
+        'node_modules/angular-touch/angular-touch.js',
+        'node_modules/angular-sanitize/angular-sanitize.js',
+        'node_modules/angular-message-format/angular-message-format.js',
+        'node_modules/angulartics/src/angulartics.js',
+        'node_modules/angular-ui-router/release/angular-ui-router.js',
+        'node_modules/angular-ui-bootstrap/src/buttons/buttons.js',
+        'node_modules/angular-ui-bootstrap/src/collapse/collapse.js',
+        'node_modules/angular-ui-bootstrap/src/dateparser/dateparser.js',
+        'node_modules/angular-ui-bootstrap/src/datepicker/datepicker.js',
+        'node_modules/angular-ui-bootstrap/src/debounce/debounce.js',
+        'node_modules/angular-ui-bootstrap/src/dropdown/dropdown.js',
+        'node_modules/angular-ui-bootstrap/src/isClass/isClass.js',
+        'node_modules/angular-ui-bootstrap/src/modal/modal.js',
+        'node_modules/angular-ui-bootstrap/src/multiMap/multiMap.js',
+        'node_modules/angular-ui-bootstrap/src/popover/popover.js',
+        'node_modules/angular-ui-bootstrap/src/position/position.js',
+        'node_modules/angular-ui-bootstrap/src/progressbar/progressbar.js',
+        'node_modules/angular-ui-bootstrap/src/stackedMap/stackedMap.js',
+        'node_modules/angular-ui-bootstrap/src/tabs/tabs.js',
+        'node_modules/angular-ui-bootstrap/src/tooltip/tooltip.js',
+        'node_modules/angular-ui-bootstrap/src/typeahead/typeahead.js',
+        'node_modules/moment/moment.js',
+        'node_modules/angular-moment/angular-moment.js',
+        'node_modules/medium-editor/dist/js/medium-editor.js',
+        'node_modules/leaflet/dist/leaflet-src.js',
+        'node_modules/angular-simple-logger/dist/angular-simple-logger.js', // Required by angular-leaflet-directive
+        'node_modules/PruneCluster/dist/PruneCluster.js',
+        'node_modules/ui-leaflet/dist/ui-leaflet.js',
+        'node_modules/leaflet-active-area/src/leaflet.activearea.js',
+        'node_modules/angular-waypoints/dist/angular-waypoints.all.js',
+        'node_modules/ng-file-upload/dist/ng-file-upload.js',
+        'node_modules/message-center/message-center.js',
+        'node_modules/chosen/chosen.jquery.js',
+        'node_modules/angular-chosen-localytics/dist/angular-chosen.js',
```

Comment:
I'm seeing an error coming from this lib when on http://localhost:3000/profile/edit.

> Error: element.chosen is not a function
> init/<@http://localhost:3000/node_modules/angular-chosen-localytics/dist/angular-chosen.js?c=f4bd0515:94:31

Reply:
(and do not get that error on master branch)